### PR TITLE
fix gatling build

### PR DESCRIPTION
**Pull request by kayvank**
**Overview**
update for rnode propose API changes

**JIRA ticket:**
https://rchain.atlassian.net/browse/RCHAIN-3730

*Notes*
update for rnode propose API changes. Changes are required to fix the broken build

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ clone:
   clone-rchain:
     image: plugins/git
     commands:
-      - git clone $${RCHAIN_REPO:-https://github.com/rchain/rchain.git} /workdir/rchain && cd /workdir/rchain
```
Comment (lukasz-golebiewski) [inline .drone.yml]: I don't think this should go into master

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# Performance Harness
+Performance Harness
```
Comment (lukasz-golebiewski) [inline README.md]: Why is this no longer a heading?

File: .drone.yml (hunk 1 above)
Comment (kayvank) [reply .drone.yml]: I in inadvertently pushed my local changes.  PTLA

File: README.md (hunk 2 above)
Comment (kayvank) [reply README.md]: PTAL

**Review by lukasz-golebiewski — APPROVED** [review]

**Review by adaszko — APPROVED** [review]

**Pull request merged**